javalin-plugins/javalin-swagger-plugin/src/main/kotlin/io/javalin/openapi/plugin/swagger/SwaggerPlugin.kt

```diff
@@ -30,6 +30,20 @@ class SwaggerConfiguration {
     var tagsSorter: String = "'alpha'"
     /** Operations sorter algorithm expression. */
     var operationsSorter: String = "'alpha'"
+    /**  Custom CSS files to be injected into Swagger HTML */
+    var customStylesheetFiles: ArrayList<String> = arrayListOf()
+    /**  Custom JavaScript files to be injected into Swagger HTML */
+    var customJavaScriptFiles: ArrayList<String> = arrayListOf()
+
+    @JvmOverloads
+    fun injectStylesheet(path: String, media: String = "screen") {
+        customStylesheetFiles.add("<link href='${path}' rel='stylesheet' media='${media}' type='text/css' />")
+    }
+
+    @JvmOverloads
+    fun injectJavaScript(path: String, type: String = "text/javascript") {
+        customJavaScriptFiles.add("<script src='${path}' type='${type}' />")
+    }
```

Comment:
I think it might be nice, if those methods would return `SwaggerConfiguration` instance. You can use `also` like that: 

```kotlin
fun method(): SwaggerConfiguration = also { 
  // code 
}
```

Reply:
Nice! Changed :-)